docs/bestpractices.md

```diff
@@ -103,4 +103,125 @@ If the changes are required to be applied immedietly, the previous inventory job
+
+```
+microk8s kubectl -n sc4snmp get services
+```
+
+This command should output similar data:
+```
+NAME                                TYPE           CLUSTER-IP       EXTERNAL-IP      PORT(S)         AGE
+snmp-redis-headless                 ClusterIP      None             <none>           6379/TCP        113s
+snmp-mibserver                      ClusterIP      10.152.183.163   <none>           80/TCP          113s
+snmp-mongodb                        ClusterIP      10.152.183.118   <none>           27017/TCP       113s
+snmp-redis-master                   ClusterIP      10.152.183.61    <none>           6379/TCP        113s
+snmp-mongodb-metrics                ClusterIP      10.152.183.50    <none>           9216/TCP        113s
+snmp-splunk-connect-for-snmp-trap   LoadBalancer   10.152.183.190   114.241.233.134   162:32180/UDP   113s
+```
+
+Check the `EXTERNAL-IP` of `snmp-splunk-connect-for-snmp-trap` and the second port number for this service. In this case 
+the full `snmp-splunk-connect-for-snmp-trap` address will be `114.241.233.134:32180`.
+
+
+In case agents send traps to the correct address, but there is still no data in the `netops` index, there might some
+issues with credentials. These errors can be seen in logs if the `snmp-splunk-connect-for-snmp-trap` pod. Logs can be 
+accessed either in Splunk in case of using [sck-otel](gettingstarted/sck-installation.md) or directly in kubernetes 
+using the terminal.
+
+#### Accessing logs via Splunk
+If [sck-otel](gettingstarted/sck-installation.md) is installed, run the following search query in Splunk 
+`index=em_logs sourcetype="kube:container:splunk-connect-for-snmp-traps" error`
+
+#### Accessing logs in kubernetes
+to access logs directly in kubernetes, first run `microk8s kubectl -n sc4snmp get pods`. This will output all pods:
+```
+NAME                                                          READY   STATUS    RESTARTS   AGE
+snmp-splunk-connect-for-snmp-worker-trap-99f49c557-j9jwx      1/1     Running   0          29m
+snmp-splunk-connect-for-snmp-trap-56f75f9754-kmlgb            1/1     Running   0          29m
+snmp-splunk-connect-for-snmp-scheduler-7bb8c79855-rgjkj       1/1     Running   0          29m
+snmp-mibserver-784bd599fd-6xzfj                               1/1     Running   0          29m
+snmp-splunk-connect-for-snmp-worker-poller-78b46d668f-59mv4   1/1     Running   0          29m
+snmp-splunk-connect-for-snmp-worker-sender-6f8496bfbf-cvt9l   1/1     Running   0          29m
+snmp-mongodb-7579dc7867-mlnst                                 2/2     Running   0          29m
+snmp-redis-master-0                                           1/1     Running   0          29m
+```
+
+Now get logs of the `snmp-splunk-connect-for-snmp-trap`:
+```
+microk8s kubectl -n sc4snmp logs pod/snmp-splunk-connect-for-snmp-trap-56f75f9754-kmlgb
+```
+
+#### Error messages for different authentication problems
+
+
+##### Wrong community string
```

Comment:
Let's follow pattern of previous issues, and have the error message as title. I think it's easier to find for someone searching for solution.